src/config/index.ts

```diff
@@ -307,6 +308,7 @@ export const UPDATE_ANY_TRUSTS_INVOLVED_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UP
 export const UPDATE_STATEMENT_VALIDATION_ERRORS_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_STATEMENT_VALIDATION_ERRORS_PAGE;
 export const UPDATE_PAYMENT_FAILED_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + PAYMENT_FAILED_PAGE;
 export const RELEVANT_PERIOD_OWNED_LAND_FILTER_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE;
+export const UPDATE_RELEVANT_PERIOD_INTERRUPT_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + RELEVANT_PERIOD_INTERRUPT_PAGE;
```

Comment:
We need to be clear about these naming conventions, I was intending to go with just `RELEVANT_PERIOD_INTERRUPT_URL` not sure if we need both prefixes

Reply:
Updated